src/ApiRequest/ApiClient.java

```diff
@@ -0,0 +1,91 @@
+package ApiRequest;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.*;
+
+import Models.Time;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+
+public class ApiClient {
+
+    private static final String API_URL = System.getenv("API_URL");
```

Comment:
May be some integrations use this SDK along with other SDKs in their system and better to name the env variables explicitly referencing YaYa, so the env variable names be better like `YAYA_API_URL`, `YAYA_API_KEY`, etc